src/NuGet.Core/NuGet.Commands/RestoreCommand/OriginalCaseGlobalPackageFolder.cs

```diff
@@ -111,7 +110,7 @@ public async Task CopyPackagesToOriginalCaseAsync(IEnumerable<RestoreTargetGraph
                             identity,
                             packageDependency,
                             versionFolderPathResolver,
-                            _request.PackageExtractionContext,
+                            originalCaseContext,
```

Comment:
why this change?

Reply:
This is returning to the past logic (https://github.com/NuGet/NuGet.Client/pull/2497/files#diff-101420c352b3f5357ea7134fe678d121R25) because the request doesn't have a `PackageExtractionContext`